### DISCUSSION TITLE
[HELP] would like to know scenes to be used for param 'prometheus_http_prefix'

### DISCUSSION THREAD

**Discussion by BlueBlue-Lee** [grafana/mimir#2640] [Help and support] [answered]

**Comment by pracucci** [comment, marked as answer]
Hi! Could you clarify a bit more your question, please?

**Reply by BlueBlue-Lee** [reply]
Sorry for my poor English...    

I want to know why mimir provides a command param `http.prometheus-http-prefix` to  set  url path under which the Prometheus api will be served.  Why not directly use prometheus http api?

**Reply by pracucci** [reply]
The Prometheus-compatible API endpoints are exposed under the prefix configured by `-http.prometheus-http-prefix` which by default is `/prometheus`. For example, the API to run a range query is at `/prometheus/api/v1/query_range` by default. You can find the documentation of all Mimir APIs [here](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/) and look for `prometheus-http-prefix` to find all API endpoints exposed under `-http.prometheus-http-prefix`.

> Why not directly use prometheus http api?

Mimir exposes other APIs too, for example config management APIs. To avoid any API endpoint collision, we prefer to keep the Prometheus API endpoints exposed under a dedicated prefix. However, if you want to expose Prometheus API endpoints under the root `/` you could just set `-http.prometheus-http-prefix=""`.

**Reply by BlueBlue-Lee** [reply]
Got it, thanks.
